components/ui/MarkdownContent/stylesheets/table.scss

```diff
@@ -67,9 +67,6 @@
       border: 0;
       vertical-align: top;
 
-      sup {
-        z-index: -1;
-      }
```

Comment:
This change is good, for internal folks it was added as a result of https://github.com/github/docs-internal/pull/34190, but the bug described there no longer exists so good to remove.